metadata_service/api/table.py

```diff
@@ -208,14 +239,28 @@ def delete(self, table_uri: str, tag: str) -> Iterable[Union[Mapping, int, None]
         :param tag:
         :return:
         """
+        args = self.parser.parse_args()
+        tag_type = args.get('tag_type', 'default')
         try:
-            self.client.delete_tag(table_uri=table_uri, tag=tag)
-            return {'message': 'The tag {} for table_uri {} '
+            self.client.delete_tag(table_uri=table_uri,
+                                   tag=tag,
+                                   tag_type=tag_type)
+            return {'message': 'The tag {} for table_uri {} with type {} '
                                'is deleted successfully'.format(tag,
-                                                                table_uri)}, HTTPStatus.OK
+                                                                table_uri,
+                                                                tag_type)}, HTTPStatus.OK
         except NotFoundException:
             return \
-                {'message': 'The tag {} for table_uri {} '
+                {'message': 'The tag {} for table_uri {} with type {} '
                             'is not deleted successfully'.format(tag,
-                                                                 table_uri)}, \
+                                                                 table_uri,
+                                                                 tag_type)}, \
+                HTTPStatus.NOT_FOUND
+        except BadgeNotInWhitelistException:
```

Comment:
I wonder if we need to validate whitelist on deletion. May be we can be flexible on deletion so that others can delete it when we dropped certain badges. WDYT?

(If we decided to keep this, message needs to be updated :)  s/added/deleted)

Reply:
will do